modules/cluster-logging-collector-log-forward-sts-cloudwatch.adoc

```diff
@@ -1,19 +1,12 @@
 // Module included in the following assemblies:
 //
-// * logging/log_collection_forwarding/configuring-log-forwarding.adoc
+// * observability/logging/log_collection_forwarding/configuring-log-forwarding.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-log-forward-sts-cloudwatch_{context}"]
 = Forwarding logs to Amazon CloudWatch from STS enabled clusters
 
-For clusters with AWS Security Token Service (STS) enabled, you can create an AWS service account manually or create a credentials request by using the
-ifdef::openshift-enterprise,openshift-origin[]
-xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[Cloud Credential Operator(CCO)]
-endif::[]
-ifdef::openshift-dedicated[]
-link:https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html[Cloud Credential Operator(CCO)]
-endif::[]
- utility `ccoctl`.
+For clusters with AWS Security Token Service (STS) enabled, you can create an AWS service account manually or create a credentials request by using the link:https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html[Cloud Credential Operator(CCO)] utility `ccoctl`.
```

Comment:
We need to swap from links that link to our internal docs to xrefs. The reason for this is that, if customers are on the portal and they click this docs.openshift.com link, they'll be redirected to docs.opc.com. this is a bad UX. These xrefs will need to be put into the associated assembly, since xrefs cannot go into modules.